runtimes/manta/src/command.rs

```diff
@@ -1,20 +1,3 @@
-// This file is part of Substrate.
-
-// Copyright (C) 2017-2021 Parity Technologies (UK) Ltd.
-// SPDX-License-Identifier: Apache-2.0
-
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// 	http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
```

Comment:
Please keep the original licence and add a few lines to claim that the additional code is under GPL.

Reply:
Oh my bad I removed it because of the `` This file is part of Substrate. `` line. Will revert.

Reply:
I don't see any GPL notice anywhere else in the code.
Can you point me to the text for it .

Reply:
Nvm found it in another repo.